### DISCUSSION TITLE
Trying to run Node BackEnd

### DISCUSSION THREAD

**Discussion by jprzimba** [vercel/community#3110] [Help] [answered]
### Summary

I'm new using Vercel and I created my application with React and Node on the backend, how do I run the backend on Vercel?
when i visit the given url after the deply i get the error

`404: NOT_FOUND
Code: NOT_FOUND
ID: gru1::flz94-1688409438411-1747d83d9206`

in the development environment the backend works perfectly. I tried to use the vercel.json file but I wasn't successful

> {
>     "build": {
>         "src": "api/index.js",
>         "use": "@vercel/node"
>     },
>     "routes": [
>         {
>             "src": "/(.*)",
>             "dest": "api/index.js"
>         }
>     ]
> }

### Steps to Reproduce

Dont know how it works

**Comment by maddsua** [comment, marked as answer]
Hi. I don't think that Vercel allows you to run a regular backend, whether it's node, go or anything else. They have a CDN to serve static content generated by a frontend framework (like Astro, Vue, Next) and a bunch of serverless functions. The latest act like small islands of server logic, but they only run on-demand, like when user tries to visit a url. You may migrate your app to Vite (the build tool, shouldn't be hard at all), to Astro (framework, supports react components with some additional cool features on top) or to Vercel's NextJS (almost the same as React but there are a few foot-shooters)

**Comment by jprzimba** [comment]
Fixed issue
My package.json in node was wrong path 
Thanks
